src/theme/components/searchBar.ts

```diff
@@ -6,23 +6,24 @@ const { defineMultiStyleConfig, definePartsStyle } =
 const SearchBar = defineMultiStyleConfig({
   baseStyle: definePartsStyle({
     display: "flex",
-    marginBottom: {
-      base: "xs",
-      md: "auto",
+    "@container (min-width: 0px)": {
+      "&": { marginBottom: "xs", flexFlow: "column nowrap" },
+      " > select": { marginBottom: "-1px" },
```

Comment:
Similar to the other PR, if this can target the element directly without the need to rely on the DOM, that'd be great.